updates/sync.js

```diff
@@ -0,0 +1,110 @@
+var _ = require('underscore'),
+	rp = require('request-promise');
+	keystone = require('keystone');
+	User = keystone.list('User');
+
+module.exports = {
+
+	//TODO Continuous sync
+	syncUsersToDb: function () {
+		var options = {
+			uri: 'https://slack.com/api/users.list?',
+			qs: {
+				token: 'xoxp-2186825377-14603377189-42848621732-2e85220052'
```

Comment:
Add a config.js file in the root or even better, put this in .env file as SLACK_TOKEN